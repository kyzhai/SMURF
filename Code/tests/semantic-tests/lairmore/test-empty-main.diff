6a7,8
> 	ID: head h : t :: [a] -> a
> 	ID: tail h : t :: [a] -> [a]
