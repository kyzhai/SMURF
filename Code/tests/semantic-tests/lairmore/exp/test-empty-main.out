Program: 
	SMain: []
Global Scope: 
	ID: main  :: []
	ID: print x :: a -> a
	ID: random  :: Int
	ID: head h : t :: [a] -> a
	ID: tail h : t :: [a] -> [a]

